exercise_0/fileMode/FileMode.cpp

```diff
@@ -0,0 +1,22 @@
+//
+// Created by emyrmak on 7/3/2019.
+//
+
+#include "FileMode.h"
+#include <iostream>
+#include <fstream>
+
+int FileMode::run(const string &path) {
+    ifstream  file;
+    file.open(path);
+    if (file) {
+        while(file >> inputText) {
```

Comment:
What is the file layout in terms of data?

Another generic comment - always try to apply as much flexible way of reading resources, as possible (i.e. inject an interface responsible for file reading manner).